webapp/channels/src/components/pricing_modal/card.tsx

```diff
@@ -117,36 +116,28 @@ export function BlankCard() {
 function Card(props: CardProps) {
     const {formatMessage} = useIntl();
     const bottomClassName = classNames('bottom', {
-        bottom__round: props.cloudFreeDeprecated && props.isCloud,
+        bottom__round: props.isCloud,
     });
 
     const contactSalesCTAClassName = classNames('contact_sales_cta', {
-        contact_sales_cta__reduced: props.cloudFreeDeprecated && props.isCloud,
+        contact_sales_cta__reduced: props.isCloud,
     });
 
-    const planBriefingContentClassName = classNames('plan_briefing_content', {
-        plan_briefing_content__reduced: props.cloudFreeDeprecated,
-    });
+    const planBriefingContentClassName = classNames('plan_briefing_content');
 
-    const planPriceRateSectionClassName = classNames('plan_price_rate_section', {
-        plan_price_rate_section__expanded: props.cloudFreeDeprecated,
-    });
+    const planPriceRateSectionClassName = classNames('plan_price_rate_section');
 
-    const planLimitsCtaClassName = classNames('plan_limits_cta', {
-        plan_limits_cta__expanded: props.cloudFreeDeprecated,
-    });
+    const planLimitsCtaClassName = classNames('plan_limits_cta');
 
-    const buildingImgClassName = classNames('building_img', {
-        building_img__expanded: props.cloudFreeDeprecated,
-    });
+    const buildingImgClassName = classNames('building_img');
```

Comment:
`classNames` is intended to add several classes. Since here the argument is just one, it is the same as putting directly the string. So let's put directly the string. (we could even consider put this in place, instead of creating all these variables, but 0/5.